### ISSUE TITLE
BuildConfig.DEBUG is  Boolean.parseBoolean("true") with release build type

### ISSUE THREAD

**Issue by funyoung** [artem-zinnatullin/qualitymatters#180]
With Android Studio, build the project and found DEBUG is Boolean.parseBoolean("true"), which might be Boolean.parseBoolean("false").

**Comment by artem-zinnatullin** [comment]
Hm, maybe you've switched build type and didn't sync gradle in Android
Studio? BuildConfig should be regenerated from scratch, I don't remember
putting "debuggable true" for release build type.

**Comment by funyoung** [comment]
You are right, BuildConfig is regenerated from scratch, and my Android Studio has sync after the build type switching. With some investigation, I found what make the magic change to make BuildConfig.DEBUG is always 'true' is bellowing line within the app/build.gradle in the 'release' section of buildTypes config:

testCoverageEnabled true

removing this line and the BuildConfig.java seems correct and DEBUG = false for release build.

then I add the line back, BuildConfig.DEBUG =  Boolean.parseBoolean("true");

Would like to double check and apply the patch if it is fine.

**Comment by artem-zinnatullin** [comment]
Huh, makes sense, lemme check on my side then!
